Confus/Confus/Game.cpp

```diff
@@ -5,14 +5,20 @@
 #include "Game.h"
 #include "Player.h"
 #include "Flag.h"
+
 #define DEBUG_CONSOLE
-#include "Debug.h"
+#include "../Common/Debug.h"
+
 
 namespace Confus
 {
     const double Game::FixedUpdateInterval = 0.02;
     const double Game::MaxFixedUpdateInterval = 0.1;
 
+	//Start score
+	int ClientTeamScore::RedTeamScore = 0;
+	int ClientTeamScore::BlueTeamScore = 0;
```

Comment:
I recommend placing these definitions in ClientTeamScore.cpp. Statics have a 'behaviour' similar to methods in that they need to be declared in the header (int RedTeamScore;) and defined in a cpp file (which is usually the one with the same name)